services/configuration/src/schema.rs

```diff
@@ -54,13 +55,51 @@ lazy_static! {
     ).into_bytes();
 }
 
-/// A functional equivalent to `Option<Vote>` used to store votes in the service schema.
+/// A enum used to represent different kinds of vote, `Vote` and `VoteAgainst` transactions.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "vote_for")]
+pub enum VotingDecision {
+    /// `Vote` transaction variant
+    Yea(Vote),
+    /// `VoteAgainst` transaction variant
+    Nay(VoteAgainst),
+}
+
+impl CryptoHash for VotingDecision {
+    fn hash(&self) -> Hash {
+        match *self {
+            VotingDecision::Yea(ref vote) => vote.hash(),
+            VotingDecision::Nay(ref vote_against) => vote_against.hash(),
+        }
+    }
+}
+
+impl StorageValue for VotingDecision {
+    fn into_bytes(self) -> Vec<u8> {
+        match self {
+            VotingDecision::Yea(vote) => vote.into_bytes(),
+            VotingDecision::Nay(vote_against) => vote_against.into_bytes(),
+        }
+    }
+
+    fn from_bytes(bytes: Cow<[u8]>) -> Self {
+        let raw_msg = RawMessage::from_vec(bytes.to_vec());
```

Comment:
I don't quite like the cloning here, but it seems that there is no way around. cc @DarkEld3r, @vldm - maybe, there is?

Reply:
I don't have any better ideas, either. 🙃